### PR TITLE
Set scheduler interval to step, and fewer warmup steps

### PR DESCRIPTION
**Pull request by woct0rdho**
If we don't set `"interval": "step"`, then the lightning trainer will run `lr_scheduler.step()` every epoch, and the warmup will take 4000 epochs.

Also I think 4000 steps are too much. The purpose of the warmup is to avoid the unstable gradients before the Adam optimizer builds stable estimations of the momentum and the 2-norm. When training LoRA with not many trainable parameters (~10M), and not very large betas (0.8, 0.9), I think 10 steps are enough, and then we can quickly see the effect of training.

### CODE REVIEW

**Comment by austin2035** [comment]
good job. Did you see the effect of training?
Also, Is your training script with low VRAM effective?
